engines/python/setup/djl_python/utils.py

```diff
@@ -51,29 +51,25 @@ def parse_input_with_formatter(
         try:
             content_type = item.get_property("Content-Type")
             input_map = decode(item, content_type)
+            _inputs, _param, is_client_side_batch[i] = _parse_inputs_params(
+                input_map, item, input_format_configs)
+            if input_format_configs.is_adapters_supported:
+                adapters_per_item, found_adapter_per_item = _parse_adapters(
+                    _inputs, input_map, item, adapter_registry)
+                adapters.extend(adapters_per_item)
+                found_adapters = found_adapter_per_item or found_adapters
```

Comment:
nit: I think `adapters.extend` and updating `found_adapters` should happen outside the try-except, to make it clearer that these state variables only get updated if we make it past the `continue` in except.

Reply:
updated.